src/System.Private.CoreLib/shared/System/Text/StringBuilder.cs

```diff
@@ -1179,6 +1179,17 @@ public StringBuilder Append(char value)
             return Append(value.ToString());
         }
 
+        internal StringBuilder AppendSpanFormattable<T>(T value, string format, IFormatProvider provider) where T : struct, ISpanFormattable, IFormattable
```

Comment:
I'd actually intended to add it to the other one as well, but apparently neglected to save that change.  I'd added them because there's no null check on value.  But, as these are internal anyway and only used with primitives right now, I've removed the constraint.